src/components/Search/SaveCancelButtons.tsx

```diff
@@ -38,7 +38,7 @@ export default function SaveCancelButtons({
         className={`cancel-save cancel-button ${cancelBtn.main} cancel-button-${searchType}`}
         role='button'
         name='cancel-button'
-        aria-label='cancel button'
+        aria-label='cancel edit'
```

Comment:
These ones don't need any aria-label, as they already have a good text. 

[Rule 1 of ARIA: don't use ARIA](https://www.deque.com/blog/top-5-rules-of-aria/)